components/internal/Card.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, BoxProps } from '@chakra-ui/react';
+
+export default function Card(rest: BoxProps) {
```

Comment:
Can we call this `props`?

Reply:
Please add JSDoc comment explaining that this is a custom component with custom styling